geopy/geocoders/opencage.py

```diff
@@ -113,11 +113,9 @@ def geocode(
                 This format is now deprecated in favor of a list/tuple
                 of a pair of geopy Points and will be removed in geopy 2.0.
 
-        :param str country: Provides the geocoder with a hint to the
-            country that the query resides in. This value will help the
-            geocoder but will not restrict the possible results to the
-            supplied country. The country code is a 3 character code as
-            defined by the ISO 3166-1 Alpha 3 standard.
+        :param str country: Restricts the results to the specified country or countries.
```

Comment:
It is not clear how one could specify multiple countries. Should it be a comma- or a space-separated list, or should it be a python list of strings?)

I suppose this is not a single occurrence of such an ambiguity in geopy, so I'll address that for all geocoders later. To keep things "pythonic", it should accept a list of strings instead of a single comma-separated string (as specified in their docs at https://opencagedata.com/api#forward-opt).